src/board.rs

```diff
@@ -58,6 +58,7 @@ impl Board {
             Event::Key(Key::Left)  => self.handle_lr(LR::Left, hit_bottom, false),
             Event::Key(Key::Right) => self.handle_lr(LR::Right, hit_bottom, false),
             Event::Key(Key::Up) | Event::Char('e') => self.rotate(hit_bottom, true),
+            Event::Char('s') => self.rotate(hit_bottom, true) && self.rotate(hit_bottom, true),
```

Comment:
1. When rotation of 90 degree is possible and 180 is not possible, it will end up with 90 degree rotation. how do you think? is it ok or awkward?
2. When 180 is possible and 90 is not possible, since it's shortcut for 180, it sounds to me that it should work but since it's twice of 90 rotation, it would not work by this implementation. 
3. when rotation of 90 degree is executed, return value would be false, but since it is changed, it should be true. the return value is used to expand time for user to press key before new block comes up.

Reply:
That's quite interesting!
Is that still right to see 180 degrees rotation as "rotation" or.. may be it would be better to think that as a different feature something like.. "vertical flip"

Then, there can be two points
1. vertical flip - see not as rotation, make the codes for vertical flip rather than using rotate function.
-> proper label would be.. `vertical flip`

3. shortcut for rotate twice - double counterclockwise rotation click as "s" and double clockwise rotation click as "r"..?
-> `rotate clockwise x2`, `rotate counterclockwise x2`

Reply:
Oh.... it is not supposed to be vertical flip...! Ok Ill think more about this